conf/messages

```diff
@@ -42,6 +43,19 @@ alreadyEnrolled.heading=Enrol with the safety & security service
 alreadyEnrolled.notice=The GBEORI you supplied is already enrolled with the safety and security service.
 alreadyEnrolled.makeADeclaration=Make an Entry Summary Declaration (ENS)
 
+## Non Organisation GGW Page
+nonOrgGgwPage.title=Enrol with the safety & security service
+nonOrgGgwPage.heading=Use a Government Gateway Organisation account
+nonOrgGgwPage.wrongTypeOfAccount=You have signed in with the wrong type of account. To enrol with safety and security you need a Government Gateway organisation account.
```

Comment:
```suggestion
nonOrgGgwPage.wrongTypeOfAccount=You have signed in with the wrong type of account. To enrol with safety and security you need a Government Gateway Organisation account.
```
- i.e. capitalisation different from content in prototype.

Reply:
As well as others below... (although the prototype itself is inconsistent with itself with the 
`If you have an organisation account, you need to:` line).